cfme/tests/cli/test_appliance_console.py

```diff
@@ -897,8 +897,36 @@ def test_appliance_ssl():
         casecomponent: Appliance
         caseimportance: medium
         initialEstimate: 1/3h
-    """
-    pass
+        testSteps:
+            1. 'cat /opt/rh/cfme-appliance/COPY/etc/manageiq/postgresql.conf.d/
+            01_miq_overrides.conf' run above command and grep to "ssl = on" string
+            2. find the content "hostssl all         all   all     md5" if version > 5.10 in
+            '/opt/rh/cfme-appliance/TEMPLATE/var/lib/pgsql/data/pg_hba.conf' if version  5.11
+             and use '/opt/rh/cfme-appliance/TEMPLATE/var/opt/rh/rh-postgresql95/lib/pgsql/
+             data/pg_hba.con' if version 5.10
+        expectedResults:
+            1. Confirm the contents
+            2. Confirm the contents
+    """
+    command = ('cat /opt/rh/cfme-appliance/COPY/etc/manageiq/postgresql.conf.d/01_miq_overrides'
+               '.conf')
+    result = appliance.ssh_client.run_command(command)
+    assert result.success, "SSL check command failed"
+    assert "ssl = on" in result.output, "ssl entry not found"
+
+    if appliance.version >= 5.11:
+        command = 'cat /opt/rh/cfme-appliance/TEMPLATE/var/lib/pgsql/data/pg_hba.conf'
+        result = appliance.ssh_client.run_command(command)
+        assert result.success
+        assert "hostssl all         all   all     md5" in result.output, (
+            "hostssl entry not found")
+    else:
+        command = ('cat /opt/rh/cfme-appliance/TEMPLATE/var/opt/rh/rh-postgresql95/lib/pgsql/'
+                   'data/pg_hba.conf')
+        result = appliance.ssh_client.run_command(command)
+        assert result.success
+        assert "hostssl all         all   all     md5" in result.output, (
+            "hostssl entry not found")
```

Comment:
Everything except the `command` is same for both 510 and 511, I would suggest using `widgetastic.utils.VersionPick` here for `command` and replace the if-else block.
E.g. - `command = VersionPick({LOWEST: "510_command", "5.11": "511_command"})`

Reply:
Or use if and else only for the `command` variable.
E.g - `command = "511_command" if appliance.version > "5.11" else "510_command"`